src/Illuminate/Database/Schema/ForeignIdColumnDefinition.php

```diff
@@ -30,11 +30,16 @@ public function __construct(Blueprint $blueprint, $attributes = [])
     /**
      * Create a foreign key constraint on this column referencing the "id" column of the conventionally related table.
      *
+     * @param  string|null  $foreignTable
      * @return \Illuminate\Support\Fluent|\Illuminate\Database\Schema\ForeignKeyDefinition
      */
-    public function constrained()
+    public function constrained($foreignTable = null)
```

Comment:
This is unfortunately a breaking change to the method signature. It's best that this is sent to master.

Reply:
There is no way to extend this class.